openapi/SwarmCommon.yaml

```diff
@@ -1016,7 +1016,7 @@ components:
       type: object
       properties:
         duration:
-          type: integer
```

Comment:
This is a some sort of a breaking change as previously integer parsing will fail on floating point number.

I am not sure how significant this is.

Reply:
since `.Seconds()` returns float64 it was needed to have a `number` here instead of `integer`

Reply:
maybe it's appropriate here to define a new "seconds" type with a nice description and example

Reply:
seconds type added